reinstall.sh

```diff
@@ -28,7 +28,7 @@ fi
 
 echo 'Installing nemo'
 if [[ "$INSTALL_OPTION" == "dev" ]]; then
-    ${PIP} install --editable ".[all]"
+    ${PIP} install --no-cache-dir --editable ".[all]"
```

Comment:
What are we adding `--no-cache-dir` here?

Reply:
Initially that file was used in the build process, I so wanted to ensure a smaller image size. But since the build now doesn't make use of that file, we should actually remove it completely from git. WDYT, @pablo-garay ?

Reply:
If it works correctly/properly, go ahead

Reply:
no you can't remove reinstall.sh, that's used by tons of devs. Let's just revert any changes to it

Reply:
thanks for the insight, @ericharper ! I reverted that commit